gematria/datasets/python/import_from_bhive.py

```diff
@@ -77,6 +77,14 @@
     'The index of the throughput value column in the input CSV file.',
 )
 
+flags.register_multi_flags_validator(
+    ['machine_code_hex_column_index', 'throughput_column_index'],
+    lambda flags_dict: flags_dict['machine_code_hex_column_index']
+    != flags_dict['throughput_column_index'],
+    'Expected machine code column and throughput column indices to be'
+    ' different',
+)
```

Comment:
The [Google style guide](https://google.github.io/styleguide/pyguide.html#210-lambda-functions) doesn't like "complex" lambdas (and it's very inclusive about what is complex).

A very common pattern for the validators is to use a decorator and a normal function:
```
@flags.multi_flags_validator(
    ['machine_code_hex_column_index', 'throughput_column_index'],
    message='Expected machine code column and throughput column indices to be different',
)
def _validate_input_columns(flags_dict):
  return flags_dict['machine_code_hex_column_index'] != flags_dict['throughput_column_index']
```